sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -304,30 +300,28 @@ public static void ProcessMultiplePartitionEvents([EventHubTrigger(TestHubName)]
 
         private Tuple<JobHost, IHost> BuildHost<T>()
         {
-            JobHost jobHost = null;
-
-            var config = new ConfigurationBuilder()
-                .AddEnvironmentVariables()
-                .AddTestSettings()
-                .Build();
-
-            const string connectionName = "AzureWebJobsTestHubConnection";
-            string connection = config.GetConnectionStringOrSetting(connectionName);
-            Assert.True(!string.IsNullOrEmpty(connection), $"Required test connection string '{connectionName}' is missing.");
-
+            JobHost jobHost;
             IHost host = new HostBuilder()
+                .ConfigureAppConfiguration(builder =>
+                {
+                    builder.AddInMemoryCollection(new Dictionary<string, string>()
+                    {
+                        { "AzureWebJobsStorage", TestEnvironment.StorageAccountConnectionString }
+                    });
+                })
                 .ConfigureDefaultTestHost<T>(b =>
                 {
                     b.AddEventHubs(options =>
                     {
                         // TODO: alternative?
                         //options.EventProcessorOptions.EnableReceiverRuntimeMetric = true;
-                        options.AddSender(TestHubName, connection);
-                        options.AddReceiver(TestHubName, connection);
+                        options.AddSender(TestHubName, TestEnvironment.EventHubsNamespaceConnectionString);
+                        options.AddReceiver(TestHubName, TestEnvironment.EventHubsNamespaceConnectionString);
                     });
                 })
                 .ConfigureLogging(b =>
                 {
+                    b.AddConsole();
```

Comment:
Logs would be part of NUnit tests output.

Reply:
Please consider making that an option;  when using the console to run tests, having a bunch of log spam in there often makes it difficult to drill in on something specific.

Reply:
Good point.